StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/SimpleMandateElement.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SimpleMandateElement.swift
+//  StripePaymentSheet
+//
+//  Created by Yuki Tokuhiro on 3/26/23.
+//
+
+@_spi(STP) import StripeUICore
+import UIKit
+
+class SimpleMandateElement: PaymentMethodElement {
+    func updateParams(params: IntentConfirmParams) -> IntentConfirmParams? {
```

Comment:
Nit:  Could we maybe add a little more comments around this logic and what it means for these params to be nil.

Reply:
Thanks for the suggestion.  As I wrote it up, I realized we should set `didDisplayMandate = true` in the case where the customer already saw the mandate too.